src/components/NewPoll/NewPollForm.js

```diff
@@ -4,18 +4,20 @@ import colours from "../../assets/colours";
 import { SelectionOfChannels } from "./SelectionOfChannels";
 import PollSubmitted from "../PollSubmitted/PollSubmitted";
 
-// Eventually we will get the channels data from our backend, here is some dummy data:
-// const channels = ["cohort9-students", "general", "help-me"];
+const API_CHANNELS =
+  process.env.NODE_ENV === "development"
+    ? process.env.REACT_APP_API_CHANNELS
+    : `http://${process.env.REACT_APP_PROJECT_NAME}-backend.bridgeschoolapp.io`;
```

Comment:
Since you already have `API_BASE_URL` in `index.js`, I would suggest `export`-ing that constant, importing it here, and then doing

```
const API_CHANNELS = API_BASE_URL + "/channels"
```